mucommander-core/src/main/java/com/mucommander/search/SearchDialog.java

```diff
@@ -248,6 +274,33 @@ private void cleanError() {
         showDialog();
     }
 
+    void addSizePanel(XAlignedComponentPanel compPanel) {
+        JPanel sp = new JPanel(new FlowLayout());
+        cboSizeRel.setSelectedIndex(lastSizeRel.ordinal());
+        sp.add(cboSizeRel);
+
+        tfSize.setText(lastSize == null ? "" : lastSize.toString());
+        sp.add(tfSize);
+
+        cboSizeUnit = new JComboBox(getSizeUnitDisplayStrings());
+        cboSizeUnit.setSelectedIndex(lastSizeUnit.ordinal());
+        sp.add(cboSizeUnit);
+
+        JPanel p = new JPanel(new BorderLayout());
+        p.add(sp, BorderLayout.WEST);
+
+        JLabel l = compPanel.addRow(Translator.get("search_dialog.size", "Size"), p, 20);
+        l.setDisplayedMnemonic('s');
+        l.setLabelFor(cboSizeRel);
+    }
+
+
+    private String[] getSizeUnitDisplayStrings() {
+        return Arrays.stream(SizeUnit.VALUES).map(s->Translator.get("search_dialog.size_unit." + s.name() , s.name()))
```

Comment:
I believe the second argument to Translator#get is not needed since there's no placeholder in the translated string, right?